### ISSUE TITLE
`else` with `try` is not detected

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1866]
We have this function:

```python
def is_literal(node) -> bool:
    try:
        ast.literal_eval(node)
    except ValueError:
        return False
    else:
        return True
```

It should raise:

```python
  5:9      WPS503 Found useless returning `else` statement
  return True
```

Because it should be consistent with `if` rule. So, the desired version is:

```python
def is_literal(node) -> bool:
    try:
        ast.literal_eval(node)
    except ValueError:
        return False
    return True
```

**Comment by sobolevn** [comment]
What about `for`? It has different `else` semantics.

**Comment by sobolevn** [comment]
Ok, here's the deal with `for`:

```python
def wrapper():
    for x in ...:
        if ...:
            break
    else:
        return 1
    return 2
```

This is not the same as `if`, but still can be checked. How?
- `else` triggers if `break` was not called and it will return `1`
- if `break` was called it will return `2`

Here's the refactored version:

```python
def wrapper():
    for x in ...:
        if ...:
            return 2
    return 1
```

**Comment by sobolevn** [comment]
The same about `while`
